core_backend/tests/api/test_manage_content.py

```diff
@@ -160,6 +160,71 @@ async def test_delete_content(
         assert response.status_code == 200
 
 
+class TestMultUserManageContent:
+    @pytest.mark.parametrize(
+        "content_title, content_text, content_metadata",
+        [("title 3", "test content 3", {})],
+    )
+    async def test_content_access_overlap(
+        self,
+        client: TestClient,
+        content_title: str,
+        content_text: str,
+        fullaccess_token: str,
+        fullaccess_token_user2: str,
+        content_metadata: Dict[Any, Any],
+    ) -> None:
+        # make content as user1
+        response = client.post(
+            "/content",
+            headers={"Authorization": f"Bearer {fullaccess_token}"},
+            json={
+                "content_title": content_title,
+                "content_text": content_text,
+                "content_language": "ENGLISH",
+                "content_metadata": content_metadata,
+            },
+        )
+        assert response.status_code == 200
+        json_response = response.json()
+        assert json_response["content_metadata"] == content_metadata
+        assert "content_id" in json_response
+
```

Comment:
Thanks for this!! May I suggest breaking down this tests however?
Each comment defines what the next block of code is testing... ideally each test would
be that block of code! Because if we fail "try and fail to fetch content as user2" we
have no idea whether the rest of the logic works or not.

I think you can use something like the `existing_content_id` fixture to achieve this.
Create a fixture like that using user 1, and reuse it for each unit test:
1. User 1 can delete it (*although I think this is tested by the TestManageContent class?)
2. User 2 can't get it
3. User 2 can't edit it
4. User 2 can't delete it